src/helpers/comment.ts

```diff
@@ -9,6 +9,7 @@ type Node = {
 
 const traverse = (result: Record<string, string[]>, node: Node, itemsToExclude: string[]): Record<string, string[]> => {
   if (itemsToExclude.includes(node.nodeName)) {
+    result[node.nodeName].push(node.nodeName);
```

Comment:
this will cause an error if nodeName is not in the object yet

https://github.com/ubiquity/ubiquibot/blob/02deb96a2021c6ef6841294d44984ed3b8792854/src/helpers/comment.ts#L21

I recommend to just adding a condition here like `!itemsToExclude.includes(node.nodeName)`